src/View/SSTemplateParser.php

```diff
@@ -4020,10 +4020,10 @@ function CacheBlock_CacheBlockTemplate(&$res, $sub)
         // Get any condition
         $condition = isset($res['condition']) ? $res['condition'] : '';
 
```

Comment:
Good point - maybe I do need to start looking for occurrences in `*.*` rather than `*.php` for these cases...